stdlib/email/base64mime.pyi

```diff
@@ -0,0 +1,7 @@
+def header_length(bytearray: str | bytes) -> int: ...
```

Comment:
found evidence in the cpython unit test that this could be a string too (it's only used as bytes in the actual code: `Lib/email/charset.py` though)
![image](https://user-images.githubusercontent.com/80432516/147148662-be0b4a2a-ce52-4a3a-a732-df63752303a6.png)